tools/features/com.liferay.ide.hidden/feature.xml

```diff
@@ -142,4 +147,59 @@
 		version="5.2.0.v20170126-0011"
 	/>
 
+	<plugin
```

Comment:
We need to create a whole new feature for this.  Call it "Liferay Language Server Extension"